x/authz/expected_keepers.go

```diff
@@ -11,6 +12,8 @@ type AccountKeeper interface {
 	GetAccount(ctx context.Context, addr sdk.AccAddress) sdk.AccountI
 	NewAccountWithAddress(ctx context.Context, addr sdk.AccAddress) sdk.AccountI
 	SetAccount(ctx context.Context, acc sdk.AccountI)
+
+	address.Codec
```

Comment:
If we are following Uber's Go style, this should be first line of the interface: https://github.com/uber-go/guide/blob/master/style.md#embedding-in-structs